tests/functional/test_basic.py

```diff
@@ -113,27 +105,30 @@ def test_codecs(pyi_builder):
 
 
 def test_compiled_filenames(pyi_builder):
-    pyi_builder.test_source("""
+    pyi_builder.test_source(
```

Comment:
Here, the opening """ were moved to the line below, as opposite of previous test cases.